test/bats/cli-test.bats

```diff
@@ -36,7 +36,10 @@ load helpers
 }
 
 @test "schemavalidator verifier test" {   
-    run bin/ratify verify -c $RATIFY_DIR/config.json -s $LOCAL_TEST_REGISTRY/schemavalidator:v0
+    # TODO 
+    # It's best to use an image with individual artifact types vs an all-in-one so any failures can be isolated.
+    # Replace this image reference once we have a local private registry for Ratify.
+    run bin/ratify verify -c $RATIFY_DIR/schemavalidator_config.json -s wabbitnetworks.azurecr.io/test/all-in-one-image:signed
```

Comment:
Is it possible to use the local registry image instead of the ACR all in one? Should we just skip this test like the K8 bats tests have done?